src/utilities/signtx.ts

```diff
@@ -0,0 +1,67 @@
+import WalletStore from "@/store/WalletStore";
+import { MyAlgoWalletSession, WebMode } from "@algo-builder/web";
+import {
+	JsonPayload,
+	WalletMultisigMetadata,
+} from "@algo-builder/web/build/algo-signer-types";
```

Comment:
Import from build

Reply:
fixed it in second PR, since it also had InitMsig tab changes